emails/templates/status/StatusEmail.ts

```diff
@@ -0,0 +1,16 @@
+import { Email } from "src/utils/types";
+import urls from "src/utils/urls";
+
+// StatusEmail is an example template
+export class StatusEmail implements Email {
+  readonly templateName = "status";
+
+  get data() {
+    return {
+      ...this.templateData,
+      urlString: urls.baseUrl, // TODO: Change to homepage URL
```

Comment:
What do you mean by homepage URL here?

Reply:
It's supposed to be a path that links to the page where users can track their projects. This is just an example template I pulled form NPP 2.0, which originally came from NPP 1.0. AKA if `localhost:3000/my-project` is where I can track the project, then it should be `/my-project`